### DISCUSSION TITLE
Welcome Emails

### DISCUSSION THREAD

**Discussion by hackmonker** [hrfee/jfa-go#79] [Q&A] [answered]
I have a custom email html but I dont know how to change the username according to the person the email is sent. Where can I see the welcome emails of jfa-go so I can understand and maybe reverse engineer it

**Comment by hrfee** [comment, marked as answer]
I added a built-in email editor in v0.3.2, which has the username in the welcome email. Could you update and remake your custom emails in it?

**Reply by hackmonker** [reply]
I tried it and it worked pefectly thanks. just for some reason its not using the custom subject that I set. does the custom subject only work for default emails?

**Reply by hackmonker** [reply]
ok checked this and found out, the welcome mail subject is using the subject for confirmation mail but that is disabled.
I even changed the confirmation mail to something random and the next mail uses that as subject

**Reply by hrfee** [reply]
The welcome email should be using the title set in settings > Welcome Emails. This works for me, even with a custom email enabled.

**Reply by xarmadigi** [reply]
> ‎Agregué un editor de correo electrónico incorporado en v0.3.2, que tiene el nombre de usuario en el correo electrónico de bienvenida. ¿Podría actualizar y rehacer sus correos electrónicos personalizados en él?‎

And... Where is the editor??

EDIT: I have seen it yet.

**Comment by hackmonker** [comment]
ok just saw the editor. smh
